ical_to_gcal_sync.py

```diff
@@ -25,7 +25,24 @@
 
 DEFAULT_TIMEDELTA = timedelta(days=365)
 
-def get_current_events():
+
+def get_current_events_from_files():
+    """Retrieves data from iCal files"""
+
+    from glob import glob
+    
+    event_ics = glob(ICAL_FEED + '*.ics')
+    
+    ics = event_ics[0]
```

Comment:
I got an exception here initially because I hadn't added a separator char to the end of my ICAL_FEED path: the glob expression ended up being "test_folder*.ics" which naturally returned an empty list and so trying to access the first element failed.

Could you tweak it slightly to avoid this happening? I guess you could do ```os.path.join(ICAL_FEED, '*.ics')``` to take care of the potentially missing separator, and then just check ```len(event_ics) > 0``` before accessing it.

Reply:
Good suggestions.  Changes committed to the branch.